lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -857,6 +857,10 @@ function dosomething_reportback_mbp_request($entity) {
       'impact_noun' => $entity->noun,
       'image_markup' => $image_markup,
     );
+    $node = node_load($entity->nid);
```

Comment:
I've been told to avoid `node_load()` due to the weight of using the function. Would be less load on the system is the Reportback entity had the campaign language?